src/main/java/net/sacredlabyrinth/phaed/simpleclans/Clan.java

```diff
@@ -1358,12 +1358,12 @@ private boolean sendBbTime(Player player, String msg) {
             long time = (System.currentTimeMillis() - Long.parseLong(msg.substring(0, index))) / 1000L;
             msg = ChatUtils.parseColors(bbPrefix + msg.substring(++index));
 
-            TextComponent textComponent = new TextComponent(msg);
+            TextComponent textComponent = new TextComponent(TextComponent.fromLegacyText(msg));
             textComponent.setHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, TextComponent.fromLegacyText(
                     Dates.formatTime(time, 1) + lang("bb.ago"))));
             player.spigot().sendMessage(textComponent);
```

Comment:
I think we can use BaseComponent[] here.

[player.spigot().sendMessage(net.md_5.bungee.api.chat.BaseComponent...)](
https://helpch.at/docs/1.15.2/org/bukkit/entity/Player.Spigot.html#sendMessage-net.md_5.bungee.api.chat.BaseComponent...)